services/src/main/java/org/apache/unomi/services/impl/definitions/DefinitionsServiceImpl.java

```diff
@@ -160,7 +170,7 @@ private void processBundleStop(BundleContext bundleContext) {
         if (bundleContext == null) {
             return;
         }
-        List<PluginType> types = pluginTypes.get(bundleContext.getBundle().getBundleId());
+        List<PluginType> types = pluginTypes.remove(bundleContext.getBundle().getBundleId());
```

Comment:
Is this a bug fix ?

Reply:
That was done by you in UNOMI-286 that I based my work on, so I can only guess :)

Reply:
ok sorry didn't realize that :)